test/File.js

```diff
@@ -909,53 +1094,60 @@ describe('File', function() {
         path: '/test/test.coffee',
       });
       file.extname = '.png';
-      file.path.should.equal('/test/test.png');
+      file.path.should.equal(path.normalize('/test/test.png'));
       done();
     });
   });
 
   describe('path get/set', function() {
```

Comment:
can you add some tests for `path` that do concatenation of the path parts and compares to the path passed in?
Basically, `file.path === file.dirname + file.basename + file.ext` for a wide array of construction types (directory with/without ending slash, files with/without extensions, etc).